tests/async_fixtures/test_async_fixtures_with_finalizer_scope.py

```diff
@@ -0,0 +1,38 @@
+import asyncio
+import contextlib
+import functools
+import pytest
+
+
+@pytest.mark.asyncio
+async def test_module_scope(port):
+    await asyncio.sleep(0.01)
+    assert port
+
+@pytest.fixture(scope="module")
+def event_loop():
+    """Change event_loop fixture to module level."""
+    policy = asyncio.get_event_loop_policy()
+    loop = policy.new_event_loop()
+    yield loop
+    loop.close()
+
+
+@pytest.fixture(scope="module")
+async def port(request, event_loop):
+    def port_finalizer(finalizer):
+        async def port_afinalizer():
+            await finalizer(None, None, None)
+        event_loop.run_until_complete(port_afinalizer())
+
+    context_manager = port_map()
+    port = await context_manager.__aenter__()
+    request.addfinalizer(functools.partial(port_finalizer, context_manager.__aexit__))
+    return True
+
+
+@contextlib.asynccontextmanager
```

Comment:
This seems to fail the build on older Python versions. I don't think it is an essential part of the test. So you might get it to run on all required versions by changing the test a bit.

Reply:
Fixed.